### ISSUE TITLE
Scene is missing property 'time' that certain functionality relies on.

### ISSUE THREAD

**Issue by MulleDK19** [ManimCommunity/manim#816]
## Description of bug / unexpected behavior
Scene does not contain property 'time', yet some functionality relies on it, for example Scene.add_sound().

## Expected behavior
Scene should have a time property containing the current time since the scene started.

## How to reproduce the issue
Try to use self.add_sound() in a scene.

<details><summary>Code for reproducing the problem</summary>

```py
class TestScene(Scene):
    def construct(self):
        self.add_sound(r"H:\somesound.wav")
        self.wait(3)
```

</details>

## Workaround.
This is a workaround I'm using currently to type write. Instead of having to add each click in Premiere manually, I'd rather use manim to add the sounds. But the time property is missing, so I had to set that manually.

```py
class TypeWrite(AddTextLetterByLetter):
    the_scene = None
    last_index = -1
    sound_paths = []

    def __init__(self, scene, text, **kwargs):
        self.the_scene = scene
        for i in range(1, 12 + 1):
            self.sound_paths.append(r"h:\CLICKS\Click_" + str(i) + r".wav")
        super().__init__(text, **kwargs)
    
    def update_submobject_list(self, index):
        self.mobject.submobjects = self.all_submobs[:index]
        if (index != self.last_index):
            self.the_scene.add_sound(random.choice(self.sound_paths))
            self.last_index = index

class TestScene(Scene):
    def construct(self):
        timer = Mobject()
        self.time = 0.0
        def t(m, dt):
            self.time = self.time + dt
        self.time = 0.0
        timer.add_updater(t)
        self.add(timer)
        
        t = Text("Type writing effect with sound, all in manim", font = "Consolas")
        self.play(TypeWrite(self, t))
```

**Comment by leotrs** [comment]
Hi @MulleDK19, thank you for your bug report.

It appears that `Scene.time` has moved to `Renderer.time`. Can you please try the following? Instead of your workaround (great job on that btw), can you try replacing the following line

https://github.com/ManimCommunity/manim/blob/1a409939aaa2131bcc18326c39cfe25d25c8d3d5/manim/scene/scene.py#L826

with this instead?
```python
time = self.renderer.time + time_offset
```

I am not able to test this locally because I don't have sound files to add to the scene :sweat_smile: But I believe it should work. If it does, can I ask you to submit a quick PR with this fix? Thanks!

**Comment by kolibril13** [comment]
I think this is solved with #904
